mongodbatlas.go

```diff
@@ -126,6 +128,56 @@ func (m *MongoDBAtlas) NewUser(ctx context.Context, req dbplugin.NewUserRequest)
 		return dbplugin.NewUserResponse{}, err
 	}
 
+	clusters, _, err := client.Clusters.List(ctx, m.ProjectID, &mongodbatlas.ListOptions{})
+	if err != nil {
+		return dbplugin.NewUserResponse{}, fmt.Errorf("roles array is required in creation statement")
```

Comment:
[nit] no need for formatting
```suggestion
		return dbplugin.NewUserResponse{}, errors.New("roles array is required in creation statement")
```